src/frontend/src/components/Post/telescope-post-content.css

```diff
@@ -47,6 +47,10 @@
   height: 1px;
 }
 
+.telescope-post-content img[src*='1.bp.blogspot.com/'] {
+  display: block;
```

Comment:
@humphd @humphd Thank you for pointing this out. I noticed that those pictures scaled beyond their natural size belong to Blogger.com posts. Therefore, to avoid messing up inline image cases. I have push another commit in which I specifically declare that any picture from Blogger.com would be `display: block`. What do you think about my code this time? I truly appreciate your enlightenment.

Reply:
Are we sure that blogger images are *never* inline?  That seems hard to believe.  Do you have some source for this info?

Reply:
Yah, you're right. I'm not completely sure the images from blogger are never inline. Hmm...